### PR TITLE
additional redirect because people are forgetting the case-sensitivit…

### PR DESCRIPTION
**Pull request by markwilkinson**
additional redirect because people are forgetting the case-sensitivity of the original AmIFAIR

### CODE REVIEW

**Comment by davidlehn** [comment]
We need to avoid making files or directories that only differ in case.  Some systems have problems with that and can't even check out the code if such things exist.  If it's really a problem then the solution is to put a fix in the top-level .htaccess.  We did that for one other project, but would prefer to avoid that when possible.  What would you like to do?

**Comment by markwilkinson** [comment]
I will back-out the request.  
Thanks!

File: certainty_nanopubs/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Redirect for the Certainty Nanopublications
+
+From the thesis of Mario Prieto come these exemplar Nanopuiblications
```
Comment (TallTed) [inline certainty_nanopubs/README.md]: typo --
`Nanopuiblications` -> `Nanopublications`

**Comment by davidlehn** [comment]
This is confusing. :-)  There's the lowercase patch, a unrelated nanopub patch with unfixed spelling issue, and a revert of the first commit.  In the future please use a fresh branch and PR for new changes.  Thanks.

**Pull request merged**